src/clib/pioc_support.c

```diff
@@ -2422,7 +2424,9 @@ inq_file_metadata(file_desc_t *file, int ncid, int iotype, int *nvars,
                     (*rec_var)[v] = 0;
 
             }
+	    PLOG((2,"varid %d var_ndims %d nunlimdims %d rec_var %d unlimdimids[0]=%d var_dimids[0]=%d\n",v,var_ndims, nunlimdims, (*rec_var)[v], unlimdimids[0], var_dimids[0]));
```

Comment:
OK, could this PLOG line of code be causing your problems? In it you reference some [0] array locations, perhaps they are not always there and that is what the memory checker is complaining about?